src/Microsoft.DotNet.ScenarioTests.SdkTemplateTests/SdkTemplateTests.cs

```diff
@@ -32,6 +32,16 @@ public void VerifyConsoleTemplateComplex(DotNetLanguage language)
         newTest.Execute(_sdkHelper, _scenarioTestInput.TestRoot);
     }
 
+    [Theory]
+    [MemberData(nameof(GetLanguages))]
+    public void VerifyConsoleTemplateAOT(DotNetLanguage language)
+    {
+        var newTest = new SdkTemplateTest(
+            nameof(SdkTemplateTests) + "AOT", language, _scenarioTestInput.TargetRid, DotNetSdkTemplate.Console,
+            DotNetSdkActions.Build | DotNetSdkActions.Run | DotNetSdkActions.PublishAOT);
+        newTest.Execute(_sdkHelper, _scenarioTestInput.TestRoot);
```

Comment:
Does this execute the test as well? We might want to print the value of `IsDynamicCodeSupported` or similar, just to prove it's Native AOT and it's running correctly.